pika/adapters/tornado_connection.py

```diff
@@ -56,13 +59,14 @@ def __init__(self, parameters=None,
 
     def _adapter_connect(self):
         """Connect to the remote socket, adding the socket to the IOLoop if
-        connected
+        connected. Set the socket non-blocking after super() call.
 
         :rtype: bool
 
         """
         error = super(TornadoConnection, self)._adapter_connect()
         if not error:
+            self.socket.setblocking(0)
```

Comment:
@wjps, what is the purpose of this `setblocking(0)` call here if your change in super `BaseConnection._adapter_connect()` already does the same?